### PR TITLE
May 17 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 74f5b19 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-457.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

File: content/en/docs/amplify_relnotes/20240517_amplify.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+* Go to [https://repository.axway.com/catalog?q=agents](https://repository.axway.com/catalog?q=agents).
+
+### Axway Agent SDK
+
+The following enhancements and bug fixes are available in this release.
+
+#### Enhancements for Agent SDK
+
+There are no new enhancements in this release.
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240517_amplify.md]: fixed

File: content/en/docs/amplify_relnotes/20240517_amplify.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+* **Usage Audit Log** - A new Usage Audit log has been enabled by default.  This Usage log will store a usage count line each time it has been successfully communicated to the Axway Platform.  The usage audit log will be stored on the local file system for at least 365 days pr imto; tje congiufration file size limit has been reached.
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240517_amplify.md]: fixed

File: content/en/docs/amplify_relnotes/20240517_amplify.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+
+* **Usage Audit Log** - A new Usage Audit log has been enabled by default.  This Usage log will store a usage count line each time it has been successfully communicated to the Axway Platform.  The usage audit log will be stored on the local file system for at least 365 days pr imto; tje congiufration file size limit has been reached.  
+
+#### Bug fixes for Agent SDK
+
+The following bug fixes are included in this release.
+
+| Case ID     | Internal ID  | Description                                       |
+|---------    |--------------|---------------------------------------------------|
+| 01567300 <br />01578477   | APIGOV-27915 <br />APIGOV-27886 | **Issue**: The Discovery and Traceability Agent were restarting with a concurrent map read and map error <br />**Resolution**: A fix was made to cuncurrent map reads and writes  |
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240517_amplify.md:64]: fixed

File: content/en/docs/amplify_relnotes/20240517_amplify.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+
+The embedded Azure Agents have been updated with the Axway Agent SDK. There are no enhancements in this release. 
+
+##### Bug fixes for Azure SaaS Agent
+
+The following bug fixes are available in this release.
+
+| Case ID     | Internal ID  | Description                                       |
+|---------    |--------------|---------------------------------------------------|
+|             | APIGOV-27892 | **Issue**: Azure API Service names were not being sanitized when stored in the API Server <br />**Resolution**: A fix was made so the API Service revision names which had capital letters were converted to lower case letters for storage in the API Server  |
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240517_amplify.md:143]: fixed

File: content/en/docs/amplify_relnotes/20240517_amplify.md [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+There are no enhancements in this release.
+
+#### Bug fixes for Asset Catalog
+
+The following bug fixes are available in this release.
+
+| Case ID     | Internal ID  | Description                                       |
+|---------    |--------------|---------------------------------------------------|
+| 01557374                   | APIGOV-27258 | **Issue**: Unable to add an asset to a product <br />**Resolution**: A fix was made to accurately display the active product releases an asset release is using  |
+| 01544107 <br />01537104    | APIGOV-26808 | **Issue**: Cannot achive an Asset Release <br />**Resolution**: The was fixed as part of the Unable to add an asset to a oroduct issue  |
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240517_amplify.md:221]: fixed

**Pull request merged**